src/parser/jobs/war/modules/PrimalProcs.tsx

```diff
@@ -0,0 +1,70 @@
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Procs as CoreProcs} from 'parser/core/modules/Procs'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SEVERITY_MISSED_PROCS = {
+	1: SEVERITY.MAJOR,
+}
+
+export class PrimalProcs extends CoreProcs {
+
+	override trackedProcs = [
+		{
+			procStatus: this.data.statuses.PRIMAL_REND_READY,
+			consumeActions: [this.data.actions.PRIMAL_REND],
+		},
+		{
+			procStatus: this.data.statuses.PRIMAL_RUINATION_READY,
+			consumeActions: [this.data.actions.PRIMAL_RUINATION],
+		},
+		{
+			procStatus: this.data.statuses.WRATHFUL,
+			consumeActions: [this.data.actions.PRIMAL_WRATH],
+		},
+	]
+
+	protected override addJobSpecificSuggestions() {
+		const missedRend = this.getDropCountForStatus(this.data.statuses.PRIMAL_REND_READY.id)
+		const missedRuination = this.getDropCountForStatus(this.data.statuses.PRIMAL_RUINATION_READY.id)
+		const missedWrath = this.getDropCountForStatus(this.data.statuses.WRATHFUL.id)
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.PRIMAL_REND.icon,
+			content: <Trans id="war.primalrend.suggestions.dropped.content">
```

Comment:
_Nit:_ The usual standard for keys is `{job}.{module}.{content type}.{specific issue}.{whatever else}`, so this (and the two below) would be better as something like `war.procs.suggestions.dropped-primal-rend.content`, etc. This also applies to the `.why` text in the suggestions.

Reply:
That make sense, will update. In keeping with that would it also be better to rename this module `Procs.tsx` so the translation keys match it cleanly?

Reply:
Doesn't make a huge difference either way. I think it most (or at least a lot of) cases, we tend to use the core name for the override and import the actual core module as `Core[ModuleName]` (like `CoreProcs` in this case), but it's not super important. The main exceptions to that rule (at least in my experience) are when you have multiple overrides for the same core module on one job (e.g. Procs.tsx and Harpe.tsx on RPR, both of which override the core Procs module), or when you have a meme filename (e.g. DirtyDancing.tsx and Technicalities.tsx on DNC).

Reply:
Gotcha, ty. The meme names are fun but I'll go with the standard and rename this to `Procs` so it's obvious that it's just doing very standard proc tracking and there isn't anything special about these 3 WAR abilities.